src/controller.c

```diff
@@ -118,10 +197,10 @@ void act_on_user_input(
             }
             break;
         case PAUSE_KEY: // pause the game
-            // TODO: @skelly pause the game
+            pause_mode();
```

Comment:
I think it would be preferable to handle all state changes by modifying the game state struct and then handling all execution branches in the `begin_game` method